oracle/feed_binance.go

```diff
@@ -151,6 +151,10 @@ func (f *binancePriceFeed) PullPrice(ctx context.Context) (
 	return priceResp.Price, nil
 }
 
+func (f *binancePriceFeed) PullAssetPair(ctx context.Context) (assetPair oracletypes.AssetPair, err error){
```

Comment:
This is pretty hacky. Better to create a separate flow for Stork

Reply:
Could you check the new flow of Stork, currently I still use the base of the core logic